testing-modules/junit-5/src/test/java/com/baeldung/groupedvsmultipleassertions/MultipleAssertionsUnitTest.java

```diff
@@ -0,0 +1,16 @@
+package com.baeldung.groupedvsmultipleassertions;
+
+import groupedvsmultipleassertions.User;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class MultipleAssertionsUnitTest {
+    @Test
```

Comment:
Why don't we put both versions of the unit test in the same test fixture, for ease of use? Also, let's show the failing test here for at least one use case - e.g. the first field failing... and maybe all three fields.